src/modules/access-wallet/hybrid/handlers/WalletLink/index.js

```diff
@@ -30,7 +30,7 @@ class WalletLinkWallet {
       'realrpcurlnotrequired',
       0
     );
-    this.connection._storage.clear();
+    this.connection?._storage.clear();
     this.connection.disconnect = () => {
       this.connection._storage.clear();
```

Comment:
Maybe also add the `this.connection?._storage.clear();` on line 35? In case the removeWallet event gets fired multiple times